packages/cli/src/commands/integration/client.ts

```diff
@@ -85,3 +86,20 @@ export async function provisionStoreResource(
     }
   );
 }
+
+export async function getStores(
```

Comment:
Not blocking but other behaviors like this typically get put in `src/util/{command-name}`. Would be happy with a follow-up PR.

Reply:
I've resolved this as well as the similar callout in #12233 in this PR. The `client.ts` file has been split apart into individual util files. Worth noting, while doing so, I also renamed this from `getStores` to `getResources`.